aldjemy/orm.py

```diff
@@ -114,7 +114,7 @@ def _extract_model_attrs(metadata, model, sa_models):
                 primaryjoin=(table.c[fk.column] == p_table.c[p_name]),
                 remote_side=p_table.c[p_name],
                 )
-            if backref:
+            if backref and not disable_backref:
```

Comment:
Can you add some test to check if this new code works?

Reply:
I'll see what I can do. I was hitting this bug when crating an alias over a many to many autogenerated through table.

Reply:
I've added a test of how I was using `aliased` and ran across this bug.